src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1232,7 +1238,7 @@ FixedwingPositionControl::control_takeoff(const Vector2f &curr_pos, const Vector
 			/* Launch has been detected, hence we have to control the plane. */
 
 			_l1_control.navigate_waypoints(prev_wp, curr_wp, curr_pos, ground_speed);
-			_att_sp.roll_body = _l1_control.nav_roll();
+			_att_sp.roll_body = _l1_control.get_roll_setpoint();
```

Comment:
Not now, but I think we should consider consolidating all of these calls eventually.